README.md

```diff
@@ -109,12 +114,49 @@ class PostData extends DataTransferObject
 }
 ```
 
-By adding doc blocks to our properties, their values will be validated against the given type; 
-and a `TypeError` will be thrown if the value doesn't comply with the given type.
+When defining typed properties, you can take advantage of the built-in types supported by PHP. These properties will be validated against the given type and a `TypeError` will be thrown if the value does not comply with it.
+
+```php
+use App\Models\Author;
+use Iterator;
+use Spatie\DataTransferObject\DataTransferObject;
 
-Here are the possible ways of declaring types:
+class PostData extends DataTransferObject
+{
+    /**
+     * Built in types:
+     */
+    public string $property;
+
+    /**
+     * Classes with their FQCN:
```

Comment:
This should be something like "`use`d classes", no?

Reply:
Whoops. Yeah, I am inclined to make that:

> Imported class or fully qualified class name:

Will commit shortly. Who knew it would be so much work just to update a README! 😄

Reply:
Resolved by 7fd5a5ab8c1514e3bb81eddceb9bd1b79a43be22